__tests__/types-tests.js

```diff
@@ -8,7 +8,8 @@ declare function BigInt(v: any): any;
 
 test('version', async () => {
     const info = await testServerQuery('query{info{version}}');
-    expect(info).toMatchObject({info: {version}})
+    console.log(info); // print out version
```

Comment:
Why log in unit test?

Reply:
To be sure of the version by checking CI logs.